### DISCUSSION TITLE
Week 4 R&R

### DISCUSSION THREAD

**Discussion by jakegros** [USU-ITLS-x265-23-summer/class-repo#100] [General]
Thoughts:
CSS Diner provides examples which helps to give some kind of context.  Going back through it, the examples definitely made more sense.  Previously, I thought it wasn't as helpful since I wasn't fully grasping it.  Persevering through it and making connections of how it fits the big picture of what I am doing in each step has started to shape my understanding.  The challenge is still being able to retain and implement all that CSS.
FreeCode Camp's instructional approach with the form of chunking the steps and providing the preview visual is super helpful.  I enjoyed this task because a lot of the concepts we've been learning started to fall into place. Perhaps that's also just a natural occurrence simply because we're regularly working through tasks and learning. I didn't feel near as lost even though there was quite a bit of new content to pick up. 
Seeing the output come together is rewarding plus the loose element of gamification allows for me to move through the lesson/activity for longer periods than just staring at code. 

As for the games, there was some odd randomness built into Elevator Saga.  I had success until challenge #4 (the level with two elevators) simply by running it several times after making changes.  Applying the changes is necessary.  
Untrusted- I passed first level.  Had no idea what to do next. I liked the theme and idea of it, just lost. I I would include a bit more technical information or example.
CodeCombat is engaging despite the pay wall. It was always relished by my students and I like the interface. 
I am still checking out the other two. 

Project updates: 
Website of my favorite topics- utilize tutorials, perhaps use iorad as an interactive tutorial?
Game- I have begun exploring relevant GutHub frameworks. It is interactive, show code below, include gifs or tutorials.
Accessibility- I started to outline things to include, as well as gathering sources, tutorials, documentation, and examples.
Portfolio- I still feel inclined to create the platform that I can start using as the Hub of all other projects for my degree and professional learning. 
There are some freeCodeCamp courses that could support these ideas.

**Comment by metasean** [comment]
>The challenge is still being able to retain and implement all that ... [_insert web development language, library, framework, tool, technology_]

The key is to learn the core concepts.  Among the core concepts, is knowing the resources for all the things that you won't use regularly enough to ever _truly_ learn, but that you still need to use.  The resources can be plain 'ol documentation, but should also include things like forums and slack groups, and even YouTube channels that simply help you quickly learn just enough, or to simply refresh your knowledge on a given technology, technique, etc.

"iorad" - TIL
